include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h

```diff
@@ -113,6 +113,25 @@ struct OpBinder {
     return failure();
   }
 
+  ParseResult f32FloatAttr(float &value, StringRef nameSuffix,
+                           float defaultValue = 0.0f) {
```

Comment:
Nice, I needed something like this for another task!

Reply:
Awesome! I'll land now so you can pull it in.